index.js

```diff
@@ -0,0 +1,44 @@
+import express from 'express';
+import swaggerJSDoc from 'swagger-jsdoc';
+import swaggerUi from 'swagger-ui-express';
+
+const app = express();
+const PORT = process.env.PORT || 3000;
+
+app.get('/health-check', (req, res) => {
+    res.json({ status: 'Server is running'});
+});
+
+const swaggerOptions = {
+    swaggerDefinition: {
+        info: {
+            title: 'Express Server API',
+            description: 'health-check endpoint',
+            version: '1.0.0',
+        },
+    },
+    apis: ['index.js'],
+};
+
+const swaggerSpec = swaggerJSDoc(swaggerOptions);
+
+app.use('/api-docs', swaggerUi.serve, swaggerUi.setup(swaggerSpec));
+
+/**
+ * @swagger
+ * /health-check:
+ *   get:
+ *     summary: Check if the server is running.
+ *     description: Returns a JSON response indicating the server is running.
+ *     responses:
+ *       200:
+ *         description: Server is running.
+ *         content:
+ *           application/json:
+ *             example:
+ *               status: Server is running
+ */
```

Comment:
Better to move this annotation prior described endpoint

```suggestion
```suggestion
/**
 * @swagger
 * /health-check:
 *   get:
 *     summary: Check if the server is running.
 *     description: Returns a JSON response indicating the server is running.
 *     responses:
 *       200:
 *         description: Server is running.
 *         content:
 *           application/json:
 *             example:
 *               status: Server is running
 */
 app.get('/health-check', (req, res) => {
    res.json({ status: 'Server is running'});
});
```
```